pp_transition.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+This script prepares the file(s) saved by PicklePersistence for v20. How to do it:
+1. Run this script *after* upgrading to v20.
+2. Enter the name of your persistence file below.
+
+WARNING: Save a backup of your pickle file(s) before running this!
+"""
+
+persistence_file_path = "ENTER YOUR PICKLE FILE NAME HERE"
+
+# Don't touch anything below this line!
+# -------------------------------------------------------------------------------------------------
+import pickle
+import telegram
+if telegram.__version__.startswith('13'):
```

Comment:
We can actually switch this to
```suggestion
if telegram.__version__.startswith('1'):
```
because the only allowed version will be 20.